### ISSUE TITLE
A11y_Azure Tools Storage Explorer_Blob Container Table_Table_ScreenReader: When the voiceover is turned on and the user uses arrow key to move across the column headers, its being announced twice.

### ISSUE THREAD

**Issue by bbarik06** [microsoft/AzureStorageExplorer#5765]
**GitHubTags:** #A11yMAS;#A11yTCS;#A11ySev2;#BM_AzureToolsStorageExplorer_May2022;#Azure Tools Storage Explorer;#MAC;#FTP;#DesktopApp;#WCAG1.3.1;#Voiceover;#AINotInScope;#External:VoiceOver;#A11ycaioh;#Closed;

## Environment Details:
Application Name: Azure Tools Storage Explorer 
Application Version: 1.24.0
Mac Version: macOS Monterey 12.3.1
Screen Reader: Voice over 

## Repro Steps:
1. Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility tester after verification of the fix.
2. Launch Azure Tools Storage Explorer.
3. Sign in to Azure.
4. Tab to 'Explorer' pane tree view.
5. Navigate to any of the file inside "Blob Containers"
6. Tab till 'Name' column header of the table.
7. Turn on voiceover and observe.

## Actual:
When the voiceover is turned on and the user uses arrow key to move across the column headers, its being announced twice. For example, voiceover reads "Content type, Column, activate to sort by this column, ascending, Content type, Column, activate to sort by this column, ascending, Content type, column 6 of 15".

The issue can be observed throughout the application. 

## Expected:
When the voiceover is turned on and the user uses arrow key to move across the column headers, it should be announced once. The voiceover should announce "Content type, Column, activate to sort by this column, ascending, column 6 of 15".

## User Impact:
Screen reader users will face difficulty perceiving user interface component.

## Attachment:
![11_Table Column headers](https://user-images.githubusercontent.com/98393966/169969297-f835775c-0038-4040-b60a-0c776be59875.gif)

**Comment by JasonYeMSFT** [comment]
I can reproduce the same bug using a minimum table element in chrome.

```html
<html>
<body>
<table>
  <thead>
    <tr>
      <th tabindex="0"> Col1 </th>
      <th tabindex="0"> Col2 </th>
    </tr>
  </thead>
</table>
</body>
</html>
```

Save the snippet into an html file and open it in a browser. Using screen reader such as Voice Over or NVDA to announce the table headers and we can see the contents being announced twice.

I this this is a bug in the screen reader. Could you verify the repro case and file a bug against them?

**Comment by JasonYeMSFT** [comment]
I have verified that our layout are semantically correct. It's an external bug in Voice Over to announce the column headers twice.

**Comment by JasonYeMSFT** [comment]
I went to the accessibility office hour and we agreed that this is an external bug to either Voice Over or Chromium.

**Comment by bbarik06** [comment]
@JasonYeMSFT A mail has been sent to log the external bug against the Voice Over, meanwhile please send the sample app for the apple team to check.

**Comment by JasonYeMSFT** [comment]
@bbarik06 Using our app to repro the bug won't be efficient. The best way for them to repro it is to download chrome and open an local html file with the code snippet I shared above. Then they can use Voice Over to navigate the page and see the bug.

**Comment by bbarik06** [comment]
Closing this bug as it was discussed in office hours call to be closed as its external.
